src/Solvers/WordSearchComponent.css

```diff
@@ -0,0 +1,26 @@
+.WordSearchGrid-input {
+  margin-bottom: 10px;
+  font-family: monospace;
+  letter-spacing: 10px;
+  min-height: 200px;
+}
+
+.WordSearchList-input {
+  margin-bottom: 10px;
+  min-height: 200px;
+}
+
+.WordSearchGrid-output {
+  font-family: monospace;
+  font-size: 16px;
+}
+
+.highlightedLetter {

```

Comment:
Can you make sure to scope all CSS classes with the class name like you did earlier in the file?  All classes are still at global scope despite looking local.